runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -1508,8 +1513,10 @@ private void streamingCommitLoop() {
           break;
         }
       }
-      commitStream.flush();
-      streamPool.releaseStream(commitStream);
+      if (commitStream) {
```

Comment:
```suggestion
      if (commitStream != null) {
```

Reply:
Oops, fixed,

I ran the following ./gradlew :beam-runners-google-cloud-dataflow-java:build does that not actually build this? Or perhaps I ran it in the wrong branch.

Reply:
You built the runner that submits the pipeline and not the worker component.

I think you meant to do `./gradlew :beam-runners-google-cloud-dataflow-java:worker:build`